sdk/support/mgmt-v2020_04_01/pom.xml

```diff
@@ -11,11 +11,11 @@
     <parent>
         <groupId>com.microsoft.azure</groupId>
         <artifactId>azure-arm-parent</artifactId>
-        <version>1.1.0</version>
+        <version>1.3.0</version>
         <relativePath>../../../pom.management.xml</relativePath>
     </parent>
     <artifactId>azure-mgmt-support</artifactId>
-    <version>1.0.0-beta</version>
+    <version>1.0.0</version>
```

Comment:
They do not need the beta?

Reply:
Yes, they want to release without beta, according to @xccc-msft 
and https://github.com/Azure/sdk-release-request/issues/184#issue-589425975